tests/storages_tests/test_journal.py

```diff
@@ -116,3 +117,23 @@ def test_pop_waiting_trial_multiprocess_safe() -> None:
                 if trial_id is not None:
                     trial_id_set.add(trial_id)
         assert len(trial_id_set) == num_enqueued
+
+
+@pytest.mark.parametrize("log_storage_type", LOG_STORAGE)
+def test_pickle_dump_and_load(log_storage_type: str) -> None:
+    if log_storage_type.startswith("redis"):
+        pytest.skip("The `fakeredis` does not support multi process environments.")
```

Comment:
I think the skip message should be changed.
```suggestion
        pytest.skip("The `JournalRedisStorage` is not pickalable.")
```